exporter/datadogexporter/translate_traces.go

```diff
@@ -371,7 +372,15 @@ func aggregateSpanTags(span pdata.Span, datadogTags map[string]string) map[strin
 	}
 
 	span.Attributes().Range(func(k string, v pdata.AttributeValue) bool {
-		spanTags[utils.NormalizeTag(k)] = tracetranslator.AttributeValueToString(v)
+		switch k {
+		case keySamplingPriority:
+			spanTags[k] = tracetranslator.AttributeValueToString(v)
+		case keySamplingRate:
+			spanTags[k] = tracetranslator.AttributeValueToString(v)
+		default:
+			spanTags[utils.NormalizeTag(k)] = tracetranslator.AttributeValueToString(v)
+		}
```

Comment:
Why do we don't want to normalize the `keySamplingRate` and `keySamplingPriority` tags? Is that because we know they don't need to be normalized?

Reply:
normalization will drop the leading `_`, which is unintended. I think in general we can simplify and improve our attribute => tag code, but for now just skipping sanitisation on these to fields unblocks us.

Reply:
Makes sense. One last question: this changes the behavior of the `_sampling_priority_v1` span tag (before, it was normalized and stored as `sampling_priority_v1`, now it will be stored as `_sampling_priority_v1`), are we sure this won't have any adverse effect?

Reply:
@ericmustin Did you discuss the last question in private? I am also wondering this

Reply:
oh weird i left a response but it didn't send? tl;dr `_ sampling_priority_v1` is an internal tag we could use in the future to control some details related to indexing, but as it's not currently being used by opentelemetry-clients or documented publicly, we don't anticipate any current users or adverse effects. Essentially this is a fix so users could use it in the future